packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1173,7 +1172,7 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
     });
   }
 
-  Action _createAction() {
+  Action _createActivateAction() {
```

Comment:
Here this is being changed to `_createActivateAction`, while in the other related classes they are maintaining `_createAction`. Is there a distinction here?